packages/workbox-google-analytics/test/browser/index.js

```diff
@@ -0,0 +1,64 @@
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import constants from '../../src/lib/constants.js';
+import IDBHelper from '../../../../lib/idb-helper.js';
+
+const STATIC_ASSETS_PATH = '/packages/workbox-google-analytics/test/static';
+const HIT_PAYLOAD = 'v=1&t=pageview&tid=UA-12345-1&cid=1&dp=%2F';
+
+describe(`workbox-google-analytics`, function() {
+  const db = new IDBHelper(constants.IDB.NAME, 1, 'QueueStore');
+  const resetDb = async () => {
+    const keys = await db.getAllKeys();
+    return Promise.all(keys.map((key) => db.delete(key)));
+  };
+
+  before(function() {
```

Comment:
Do you need this cleanup, and the earlier IDB cleanup? If not, I'd recommend removing it.

Reply:
I'm not sure about the `cleanState()` stuff, I just copied that from another test :) so @gauntface can probably better answer that.

As for the IDB cleanup, yes that's needed.